### PR TITLE
fix: Update to resolve issue 99

### PR DESCRIPTION
**Pull request by joseenriquevasquez**
https://github.com/dunglas/doctrine-json-odm/issues/99

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "dunglas/doctrine-json-odm",
+  "name": "joseenriquevasquez/doctrine-json-odm",
```
Comment (dunglas) [inline composer.json:2]: To be reverted.

File: src/Serializer.php [hunk 2]
```diff
@@ -16,7 +16,7 @@ final class Serializer extends BaseSerializer
     private const KEY_TYPE = '#type';
     private const KEY_SCALAR = '#scalar';
 
-    public function normalize($data, $format = null, array $context = [])
+    public function normalize($data, $format = null, array $context = []):array
```
Comment (dunglas) [inline src/Serializer.php:19]:
```suggestion
    public function normalize($data, $format = null, array $context = []): array
```

File: src/Serializer.php [hunk 3]
```diff
@@ -29,7 +29,7 @@ public function normalize($data, $format = null, array $context = [])
         return $normalizedData;
     }
 
-    public function denormalize($data, $class, $format = null, array $context = [])
+    public function denormalize($data, $class, $format = null, array $context = []):array
```
Comment (dunglas) [inline src/Serializer.php:32]:
```suggestion
    public function denormalize($data, $class, $format = null, array $context = []): array
```

**Review by dunglas — CHANGES_REQUESTED** [review]

**Comment by ddeboer** [comment]
Opened a new PR #102 to speed things up a bit.

**Comment by dunglas** [comment]
Closing in favor of #102

**Pull request closed without merging**